Controller/SuggestionProviderController.php

```diff
@@ -86,4 +103,38 @@ public function addSuggestionProvider(SuggestionProviderInterface $matcher, $gro
     {
         $this->suggestionProviders[] = array('provider' => $matcher, 'group' => $group);
     }
+
+    /**
+     * You can define your templates for each format in the bundle's configuration.
+     *
+     * @param string $format
+     *
+     * @return string
+     */
+    private function getTemplateForSuggestions($format = 'html')
+    {
+        if (!isset($this->configuration['templates'][$format])) {
+            throw new InvalidConfigurationException(
+                sprintf(
+                    'No template for format %s defined. Use one of %s or configure one for your format.',
```

Comment:
i don't think that "Use one of %s" makes sense. I would say "Only the formats %s are defined"